ios/Exponent/Supporting/Info.plist

```diff
@@ -70,6 +71,8 @@
 	</dict>
 	<key>FacebookAdvertiserIDCollectionEnabled</key>
 	<false/>
+	<key>FacebookAppID</key>
+	<string>1696089354000816</string>
```

Comment:
Is it a problem that we don't provide a `FacebookAppName`?

Reply:
It doesn't look like `FacebookDisplayName` is used much, if it all. our current value for it is `exp-test-1`, so I believe we should change this to `Expo Client` if we do want to include it